pkg/inventory/inventory.go

```diff
@@ -110,7 +113,7 @@ func SplitUnstructureds(objs []*unstructured.Unstructured) (*unstructured.Unstru
 		}
 	}
 	if len(invs) == 0 {
-		return nil, resources, NoInventoryObjError{}
```

Comment:
I think `NoInventoryObjError` can be deleted as it's only used in one more place (and a test), which should be cleaned up too.